scripts/check_licenses.py

```diff
@@ -0,0 +1,92 @@
+"""Check if license fields are valid in all records."""
+
+import asyncio
+import json
+import logging
+import os
+import pathlib
+import time
+
+VALID_LICENSE_IDENTIFIERS = [
+    "CC0-1.0",
+    "GPL-3.0-only",
+    "MIT",
+    "Apache-2.0",
+    "BSD-3-Clause",
+]
+
+logging.basicConfig(level=logging.INFO, format="[%(levelname)s] %(message)s")
+
+
+async def validate_file(path: pathlib.Path) -> int:
+    """Validate a single file."""
+    checks = 0
+    records = await asyncio.get_event_loop().run_in_executor(
+        None, lambda p: json.loads(open(p, "rb").read()), path
+    )
+
+    for record in records:
+        if rec_licenses := record.get("license"):
+            try:
+                attr = rec_licenses["attribution"]
+            except KeyError:
+                recid = record.get("recid", "UNSET")
+                message = f"License field set but without attribution in file {path.name} with recid {recid}!"
+
+                logging.error(message)
+                raise ValueError(message)
+
+            if attr not in VALID_LICENSE_IDENTIFIERS:
+                recid = record.get("recid", "UNSET")
+                message = f"Invalid license `{attr}` identifier in file {path.name} with recid {recid}! "
```

Comment:
<!-- TODO's are small, trivial, but necessary changes. Distinguishing todo comments from issues: or suggestions: helps direct the reader's attention to comments requiring more involvement.  -->
**todo:** ‏The order "invalid license FOO identifier" could be inverted for readability: "invalid license identifier FOO".

**todo:** Instead of "with recid RECID", we should say "for recid RECID".